### PR TITLE
Add `on_copy` event when `on_double_tap` dispatch and `allow_copy` flag to `MDLabel`

### PR DESCRIPTION
**Pull request by Neizvestnyj**
```py
from kivymd.app import MDApp
from kivymd.uix.label import MDLabel

from kivy.lang.builder import Builder

KV = """
<CopyLabel@MDLabel>
    halign: 'center'
    allow_copy: True
    md_bg_color: (0, 0.5, 0, 1)
    on_copy: print(f'Copied {self.text}')

ScrollView:    
    MDBoxLayout:
        padding: dp(10)
        spacing: dp(10)
        adaptive_height: True
        
        MDBoxLayout:
            orientation: 'vertical'
            padding: dp(10)
            spacing: dp(50)
            adaptive_height: True
            
            CopyLabel:
                text: 'Text to copy 1'
                adaptive_height: True
            
            CopyLabel:
                text: 'Text to copy 2'
                adaptive_height: True
            
            CopyLabel:
                text: 'Text to copy 3'
                adaptive_height: True
                
        CopyLabel:
            text: 'Text to copy 4'
        
"""


class TestApp(MDApp):
    def build(self):
        return Builder.load_string(KV)


TestApp().run()
```

### CODE REVIEW

File: kivymd/uix/label/label.py [hunk 1]
```diff
@@ -322,6 +323,22 @@ def _set_text(self, value):
...
 
+    allow_copy = BooleanProperty(False)
+    """
+    Label allow copy flag.
+
+    :attr:`allow_copy` is an :class:`~kivy.properties.BooleanProperty`
+    and defaults to `None`.
+    """
+
+    on_copy = ObjectProperty()
```
Comment (HeaTTheatR) [inline kivymd/uix/label/label.py]: This is bug!

Comment (Neizvestnyj) [reply kivymd/uix/label/label.py]: Of course)

**Comment by HeaTTheatR** [comment]
@Neizvestnyj Maybe it's better to use a double click instead of a long touch?

**Comment by Neizvestnyj** [comment]
> @Neizvestnyj Maybe it's better to use a double click instead of a long touch?

I dont think so, users alway pinch the widget before copying

**Comment by HeaTTheatR** [comment]
> > @Neizvestnyj Maybe it's better to use a double click instead of a long touch?
> 
> I dont think so, users alway pinch the widget before copying

Double-clicking on any system highlights the text. Not a long press.

**Pull request merged**